charts/stable/nextcloud/templates/_initPerms.tpl

```diff
@@ -0,0 +1,28 @@
+{{- define "nextcloud.init.perms" -}}
+{{- $uid := .Values.securityContext.container.runAsUser -}}
+{{- $gid := .Values.securityContext.container.runAsGroup -}}
+{{- $path := .Values.persistence.data.targetSelector.main.main.mountPath }}
+enabled: true
+type: {{ ternary "init" "install" .Values.nextcloud.general.fix_data_permissions }}
+imageSelector: image
+securityContext:
+  runAsUser: 0
+  runAsGroup: 0
+  runAsNonRoot: false
+  capabilities:
+    add:
+      - DAC_OVERRIDE
+      - FOWNER
+      - CHOWN
+command: /bin/sh
+args:
+  - -c
+  - |
+    echo "Setting permissions to 700 on data directory [{{ $path }}] (recursively) ..."
+    chmod 770 -R {{ $path }}
```

Comment:
This does not have to be recursive, it just checks the folder and it's overzaelus at that.
Our normal permissions shizzle would handle this...

Reply:
also needs to be non-failing, due to SCALE not accepting this cleanly.
Preferably refer to our SCALE compatible tools with our alpine image as well.